frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -139,7 +139,7 @@ export const valuesToApiFormat = (
     const parcelApiValues = landValuesToApiFormat({ data: p } as any);
     if (!!agencyId && p.agencyId === '') {
       parcelApiValues.agencyId = agencyId;
-    } else {
+    } else if ((parcelApiValues.agencyId as any).value) {
```

Comment:
yeah, don't do this unless we are sure that the agencyId has a .value field.

Reply:
Why is `agencyId` an object rather than a number?

Reply:
Tech debt related to @jdtoombs new agency field, they agencyId is saved as an object instead of a number.